__tests__/app.test.js

```diff
@@ -0,0 +1,27 @@
+const request = require("supertest");
+const app = require("../app.js");
+const seed = require("../db/seeds/seed.js");
+const data = require("../db/data/test-data/index.js");
+const db = require("../db/connection.js");
+beforeEach(() => {
+  return seed(data);
+});
+
+afterAll(() => {
+  return db.end();
+});
+
+describe("Get api topics", () => {
+  test("GET - status: 200 -should respond with an array of topics object", () => {
```

Comment:
This test could be improved: 
Currently it assumes that there will always be two or more topics. 
You have asserted that the first topic has as slug and the second topic has a description.

However the test description, rightly, describes that it should respond with an array of topic objects. 

Can you think of a more robust way of testing that every element, of an array of unknown length, is a topic object?